.env

```diff
@@ -9,7 +9,7 @@ POSTGRES_HOST=usaspending-db
 POSTGRES_PORT=5432
 
 # POSTGRES_CLUSTER_DIR should point to a path outside, or it will be mounted w/Docker
-POSTGRES_CLUSTER_DIR=../docker_db
+POSTGRES_CLUSTER_DIR=../../../data/usaspending/docker/usaspending-db
```

Comment:
Don't think these belong in the PR if they aren't in master

Reply:
wow you all are quick. Yep, i was about to remove those

Reply:
timing was perfect for me, finished lab and lunch with more time left in break